### DISCUSSION TITLE
Possibility for using variables in configfile

### DISCUSSION THREAD

**Discussion by DomHof84** [stryker-mutator/stryker-net#2422] [Q&A] [answered]
Hello,

is there a way to use variables in a configfile, e.q. for the solution filename?
Our solution filename changes every quarter with the year and quarter identifier in it.
As it is still using .net framework i have to define the solution parameter.

So i would have to change the configfiles every quarter in every testproject to the new solution name.

Is there a way on how i could tell stryker the filename with a variable? Or maybe ist there any other ways im not thinking about right now?

thanks in advance for any suggestions

cheers, Dominic

**Comment by rouke-broersma** [comment, marked as answer]
You can use the command line option instead of the config file: https://stryker-mutator.io/docs/stryker-net/configuration/#solution-path

Also, your sln needs to be renamed every quarter? Yikes!
